screengrab/lib/screengrab/runner.rb

```diff
@@ -187,16 +187,14 @@ def install_apks(device_serial, app_apk_path, tests_apk_path)
 
     def uninstall_apks(device_serial, app_package_name, tests_package_name)
       UI.message 'Uninstalling app APK'
-      apk_uninstall_output = run_adb_command("adb -s #{device_serial} uninstall #{app_package_name}",
-                                             print_all: true,
-                                             print_command: true)
-      UI.user_error! "App APK could not be uninstalled" if apk_uninstall_output.include?("Failure [")
+      run_adb_command("adb -s #{device_serial} uninstall #{app_package_name}",
+                      print_all: true,
+                      print_command: true)
 
       UI.message 'Uninstalling tests APK'
-      apk_uninstall_output = run_adb_command("adb -s #{device_serial} uninstall -r #{tests_package_name}",
-                                             print_all: true,
-                                             print_command: true)
-      UI.user_error! "Tests APK could not be uninstalled" if apk_uninstall_output.include?("Failure [")
+      run_adb_command("adb -s #{device_serial} uninstall -r #{tests_package_name}",
```

Comment:
The option `-r` is wrong. This code is a copy-and-paste from the function that install the APKs. In `install` command, the option `-r` is to reinstall the APK keeping the data. In `uninstall` command, the option `-r` doesn't exists. I will amend my commit fixing this.